confluent_server/confluent/xcclient/attributes.py

```diff
@@ -0,0 +1,93 @@
+###############################################################################
+# IBM(c) 2019 EPL license http://www.eclipse.org/legal/epl-v10.html
+###############################################################################
+# -*- encoding: utf-8 -*-
+
+
+# 'node', which can be considered a 'system' or a 'vm'
+nodeattrs = {
+    'groups': 'nodelist.groups',
+    'collective.manager': 'nodehm.mgt',
+    'discovery.passwordrules': None,
+    'discovery.policy': ['switch', 'mtms', 'sequential'],
+    'info.note': 'nodelist.comments',
+    'location.room': 'nodepos.room',
+    'location.row': 'nodepos.comments', 
+    'location.rack': 'nodepos.rack',
+    'location.u': 'nodepos.u',
+    'console.logging': 'site.consoleondemand',
+    'console.method': 'nodehm.cons',
+    'virtualization.host': 'vm.host',
+    'hardwaremanagement.manager': 'ipmi.bmc',
+    'hardwaremanagement.method': 'nodehm.mgt',
+    'enclosure.bay': None,
+    'enclosure.extends': None,
+    'enclosure.manager': None,
+    'id.model': 'vpd.mtm',
+    'id.serial': 'vpd.serial',
+    'id.uuid': 'vpd.uuid',
+    'net.bootable': None, 
+    'net.ipv4_gateway': None,
+    'net.hwaddr': 'mac.mac',
```

Comment:
Though you may not want to model this, it is a good time to highlight that 'net' attributes are special, as they can be sub-instanced to model multiple interfaces:
# nodeattrib d3 net*switch
d3: net.compute.switch: computesw1
d3: net.mgt.switch: mgtsw1
d3: net.switch:

Contrast with the mac model where multiple macs are crammed into a single field.